content/components/selectpanel.mdx

```diff
@@ -0,0 +1,222 @@
+  src="https://user-images.githubusercontent.com/40274682/228338168-a9424773-531e-41e3-8d95-261eb971c340.png"
+/>
+<Caption>
+  An optional control slot is available in the Select panel’s bottom left corner. Avoid use if not absolutely necessary
+  in order to avoid component and decision overwhelm.
+</Caption>
+
+### Anchoring
+
+Select panel can be anchored to the trigger, or centered within the page. If it’s centered within the page it should have a backdrop.
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <img
+      src="https://user-images.githubusercontent.com/40274682/228386690-dd175879-0dd6-4798-8fb5-b0ee096eceea.png"
+      alt=""
+    />
+    <Caption>Anchored Select panel to a trigger.</Caption>
+  </div>
+  <div>
+    <img
+      src="https://user-images.githubusercontent.com/40274682/228386689-ea191d76-3a50-41d9-98f0-9dc997caf4bf.png"
+      alt=""
+    />
+    <Caption>Select panel centered with a darker backdrop.</Caption>
+  </div>
+</Box>
+
+## Best practices
+
+### Be predictable
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt="Select panel with a back button, close button, Heading, subheading, five multiple choice options with three of them selected, and apply and cancel buttons."
+      src="https://user-images.githubusercontent.com/40274682/228386683-20e7ee4e-0d1b-4b6c-9dd1-eddd277b5512.png"
+    />
+    <Caption>If there are more than single options in a menu, have a user confirm their choices.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/40274682/228386685-4699374e-5f98-4dd4-bab8-f488ad0fff8a.png"
+    />
+    <Caption>
+      Never force users to submit their multi-select options by closing the dialog or auto-updating their choices upon
+      selection.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Click-to-dismiss behavior
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a clicking action on the apply button."
+      src="https://user-images.githubusercontent.com/40274682/228388751-12c1f46e-aee7-4e03-8899-b273d31f7d1b.png"
+    />
+    <Caption>
+      Guide users to save or cancel their selections to dismiss Select panel to prevent confusion and data loss.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a click action outside of the dialog."
+      src="https://user-images.githubusercontent.com/40274682/228388749-445619e6-91fa-4f9f-b300-6e1fe21c62a5.png"
+    />
+    <Caption>
+      Don’t allow a click-outside-to-dismiss behavior for Select panel. Users may be used to closing and submitting the
+      information. Doing so would result in a loss of their changes.{' '}
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+## Behavior
+
+### Opening
+
+<img
```

Comment:
Since you're referencing the `enter` key, I think it would make sense to replace `Return` with `Enter` in the image to align language.

Reply:
I concur; `Enter` is the preferred key name here.

Reply:
Got it, I'll fix. We should update some Figma components then too.